### PR TITLE
custom soup and salads also deepfrying fix

### PR DESCRIPTION
**Pull request by dwasint**
# About The Pull Request
https://github.com/AetherStation/AetherStation13/pull/99

Fixes deep fried foods not taking carrying over reagents, fixes custom soups not existing and also not taking reagents from the bowl.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog

:cl:
add: Re-added custom soups.
fix: Deep frying correctly transfers food reagents again.
/:cl:

### CODE REVIEW

File: code/datums/components/customizable_reagent_holder.dm [hunk 1]
```diff
@@ -103,6 +103,7 @@
 /datum/component/customizable_reagent_holder/proc/customizable_attack(datum/source, obj/ingredient, mob/attacker, silent = FALSE, force = FALSE)
 	SIGNAL_HANDLER
 
+	message_admins("test")
```
Comment (nednaZ) [inline code/datums/components/customizable_reagent_holder.dm:106]: Test message

**Review by nednaZ — CHANGES_REQUESTED** [review]
Just get rid of the debug message, everything else seems fine

File: code/datums/components/customizable_reagent_holder.dm (hunk 1 above)
Comment (dwasint) [reply code/datums/components/customizable_reagent_holder.dm:106]: removed

**Review by nednaZ — APPROVED** [review]
Debug message removed, looks fine to me

**Pull request merged**